### PR TITLE
Styling/grid displays

### PR DESCRIPTION
**Pull request by VictoriaFC**
## Summary:
- Add styling to quote cards on both the home and favorites page. 

## Type of Changes:
- [ ] Bug fix
- [ ] Refactor
- [x] New feature

## How was this tested:
Chrome dev tools

### CODE REVIEW

File: src/App/App.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ const App = () => {
 
         <Route path="/home" >
 					<Nav />
+						<h2 className="home-header">✨It's time to get Zen in Ten✨</h2>
```
Comment (CorCanavan) [inline src/App/App.tsx:27]: LOVE this touch! ✨

**Review by tfoxcollis — APPROVED** [review]
Looks good!  Thanks!

**Review by CorCanavan — APPROVED** [review]
Awesome styling additions! Love the icons and logos.

**Review by abodnar1 — COMMENTED** [review]
Love the additions of the icons and styling! Shouldn't affect our tests either!

**Pull request merged**